src/jobs/models/database/sequelize/sequelizeConnector.js

```diff
@@ -108,9 +108,13 @@ async function updateJob(jobId, jobInfo) {
         max_virtual_users: jobInfo.max_virtual_users,
         proxy_url: jobInfo.proxy_url,
         debug: jobInfo.debug,
-        enabled: jobInfo.enabled
+        enabled: jobInfo.enabled,
+        notes: jobInfo.notes,
+        emails: jobInfo.emails ? jobInfo.emails.map(emailAddress => {
```

Comment:
I think you were right, and webhooks are not supported in the code.

Reply:
They actually are supported, look litlte bit lower on line 157, you can see there is setWebhooks.

Reply:
Hi yes, you are right,
but for some reason when i save more than one webhook, it only shows one in the form

<img width="420" alt="Screen Shot 2020-10-15 at 16 31 56" src="https://user-images.githubusercontent.com/11491327/96134653-fe422b80-0f03-11eb-9099-0d9fd0264243.png">
<img width="376" alt="Screen Shot 2020-10-15 at 16 31 52" src="https://user-images.githubusercontent.com/11491327/96134665-ff735880-0f03-11eb-861c-d4acc8157546.png">

Reply:
Confirmed, there was a bug that cause only one webhook to show, fixed.